docker-compose.yaml

```diff
@@ -28,3 +28,9 @@ services:
     volumes:
       - ./docker/mongo/init_data:/init_data
       - ./docker/mongo/mongo_import.sh:/docker-entrypoint-initdb.d/mongo_import.sh
+
+  redis_blackbox:
```

Comment:
why not just `redis`, following the convention from the rest of this file? We're giving it a different container_name anyway, so it shouldn't conflict with anything.